### PR TITLE
fix(DDI-924): allow for dynamic data-bindinging to radio and dropdown…

### PR DESCRIPTION
**Pull request by chrisolsen**
… components

### CODE REVIEW

File: libs/web-components/src/components/dropdown/Dropdown.spec.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ describe('GoADropdown', () => {
     const items = ["red", "blue", "orange"];
     const result = render(GoADropdown, {
       testid: 'test-id',
-      name: 'favcolor',
+      name: 'favcolor2',
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.spec.ts:16]: It doesn't look like this test cares about the `name` attribute, so just wondering why did we change this?

File: libs/web-components/src/components/dropdown/Dropdown.svelte [hunk 2]
```diff
@@ -41,15 +41,43 @@
...
+        parseValues();
+        bindContext();
+
+        clearInterval(fn);
+      }
+      if (attempts > maxAttempts) {
+        console.error("goa-dropdown: missing the required `name` attribute. It must match the children's name attribute.")
+        clearInterval(fn);
+      }
+    }, 10);
```
Comment (ArakTaiRoth) [inline libs/web-components/src/components/dropdown/Dropdown.svelte:60]: What's the time between intervals?

**Review by ArakTaiRoth — CHANGES_REQUESTED** [review]

File: libs/web-components/src/components/dropdown/Dropdown.svelte (hunk 2 above)
Comment (chrisolsen) [reply libs/web-components/src/components/dropdown/Dropdown.svelte:60]: 10ms

Comment (ArakTaiRoth) [reply libs/web-components/src/components/dropdown/Dropdown.svelte:60]: Of course, I'm such a dunce, sorry.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**